.github/workflows/build.yml

```diff
@@ -53,10 +53,10 @@ jobs:
       - name: Build
         run: mvn clean install -pl -primefaces-integration-tests-jakarta -T1C -DskipTests -Dcheckstyle.skip -Djsdoc.skip.typedoc=true -Dmaven.javadoc.skip=true --batch-mode --show-version
       - name: Integration Tests (serial)
```

Comment:
We do not need IT´s in serial order any more when we target Java 11+. (This was only necessary for Java 8 because of specific Java 8 internals.)

Reply:
Good find I have removed the serial testing only parrallel now.